docs/source/markdown/podman-pod-logs.1.md

```diff
@@ -0,0 +1,91 @@
+% podman-pod-logs(1)
+
+## NAME
+podman\-pod\-logs - Displays logs for pod with one or more containers
+
+## SYNOPSIS
+**podman pod logs** [*options*] *pod*
+
+## DESCRIPTION
+The podman pod logs command batch-retrieves whatever logs are present with all the containers of a pod. Pod logs can be filtered by container name or id using flag **-c** or **--containername** if needed.
+
+## OPTIONS
+
+#### **--container**, **-c**
+
+By default `podman pod logs` retrives logs for all the containers available within the pod differentiate by field `container`. However there are use-cases where user would want to limit the log stream only to a particular container of a pod for such cases `-c` can be used like `podman pod logs -c ctrNameorID podname`.
+
+#### **--follow**, **-f**
+
+Follow log output.  Default is false.
+
+Note: If you are following a pod which is removed `podman pod rm`, then there is a
+chance the the log file will be removed before `podman pod logs` reads the final content.
+
+#### **--latest**, **-l**
```

Comment:
Does this make sense? I think `podman logs` already has a `--latest` flag, so this is duplicating existing functionality. This should either choose the latest pod, or should not exist.

Reply:
@mheon So the idea is if multiple pods are created user could just use `podman pod logs --latest` , it gets the latest pod from context.

Reply:
That's not what the description says it's doing - the description seems to imply it's choosing only one container.

If it actually is choosing the latest pod, that's fine, just need to make this a bit clearer.